packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Server/EnvVars/elements.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import EnvIcon from 'react-icons/lib/go/key';
+import Input from '@codesandbox/common/lib/components/Input';
+
+export const EnvironmentIcon = styled(EnvIcon)`
+  margin-right: 0.6rem;
+  font-size: 1rem;
+  margin-left: 4px;
+`;
+
+export const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const InputContainer = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  margin-bottom: 0.5rem;
+`;
+
+export const ErrorMessage = styled.div`
+  color: ${props => props.theme.red};
```

Comment:
Same

Reply:
In the theme I'm not gonna add it because we don't type it anyway